compiler/resolution/visibleFunctions.cpp

```diff
@@ -449,131 +469,161 @@ static void lookAtTypeFirst(const char* name, CallExpr* call,
   }
   Type* t = typeActual->getValType();
 
+  BlockStmt* typeScope = getVisibilityScope(t->symbol->defPoint);
+  // If we are already in the scope where the type is defined, we need to
+  // indicate that.  However, if we are in a different scope (even if we're in a
+  // subscope of where the type is defined), then we should treat this check as
+  // though we are following an explicit use statement, and not follow private
+  // use and import statements.
```

Comment:
This comment is just a copy of the one above, maybe write something shorter this time?
Either way I find this comment more confusing than the code, personally.

Reply:
How does it look now?

Reply:
sounds good